config/default.yaml

```diff
@@ -119,6 +119,15 @@ executor:
 
         # Launcher container tag to use
         launchVersion: stable
+    jenkins:
+      # Configuration of Jenkins
+      jenkins:
+          host: jenkins.default
+          port: 8080
+          username: screwdriver
+          password: screwdriver
```

Comment:
Can you put something more random/obnoxious here (like `WOW-AN-EVEN-MORE-INSECURE-PASSWORD!!!!`) so people know it needs to be changed